logging/runtime/src/main/java/io/quarkiverse/googlecloudservices/logging/runtime/LoggingConfiguration.java

```diff
@@ -74,6 +81,14 @@ public class LoggingConfiguration {
     @ConfigItem
     public StructuredConfig structured;
 
+    /**
+     * Configures if logs should be written to stdout or stderr instead of using Google Cloud Operations API.
+     * Useful if app is deployed to managed Google Cloud Platform environment with installed logger agent.
+     * Possible values: STDOUT, STDERR and CLOUD_LOGGING.
+     */
+    @ConfigItem(defaultValue = "STDOUT")
+    public LogTarget logTarget;
```

Comment:
Default should be CLOUD_LOGGING as it was the default before this PR.

Reply:
of course, will change :)